src/plots/attributes.js

```diff
@@ -129,6 +129,19 @@ module.exports = {
         ].join(' ')
     },
     hoverlabel: fxAttrs.hoverlabel,
+    hovertemplate: {
+        valType: 'string',
+        role: 'info',
+        dflt: false,
```

Comment:
We should make `dflt: ''` so that the default and the valid values are all `typeof string`. Luckily in JS `Boolean('') //=> false` , so you won't have to update all the `if(hovertemplate)` statements.

Reply:
Fix in 8e8b75b